src/MOI_wrapper.jl

```diff
@@ -431,5 +433,81 @@ end
     end
     # qmatind is ::Vector{Cint}, so we convert back to ::Vector{Int}.
     return sparse(Int.(qmatind), qmatcol, qmatval)
- end
+end
+
+function compute_conflict(model::Optimizer)
+    model.conflict = c_api_getconflict(model.inner)
+    return
+end
+
+"""
+    ConflictStatus()
+
+An optimizer attribute indicating the status of the last computed conflict.
```

Comment:
```
"""
Return an `MOI.TerminationStatusCode` indicating the status of the last computed conflict.

If a minimal conflict is found, it will return `MOI.OPTIMAL`. If the problem is feasible, it will
return `MOI.INFEASIBLE`.
"""
```